wideq/core.py

```diff
@@ -20,6 +21,47 @@
 DATE_FORMAT = '%a, %d %b %Y %H:%M:%S +0000'
 
 
+def get_wideq_logger() -> logging.Logger:
+    level = logging.INFO
+    fmt = "%(asctime)s %(levelname)s [%(name)s] %(message)s"
+    datefmt = "%Y-%m-%d %H:%M:%S"
+    logger = logging.getLogger("wideq")
+    logger.setLevel(level)
+
+    try:
+        import colorlog  # type: ignore
+        colorfmt = f"%(log_color)s{fmt}%(reset)s"
+        handler = colorlog.StreamHandler()
+        handler.setFormatter(
+            colorlog.ColoredFormatter(
+                colorfmt,
+                datefmt=datefmt,
+                reset=True,
+                log_colors={
+                    "DEBUG": "cyan",
+                    "INFO": "green",
+                    "WARNING": "yellow",
+                    "ERROR": "red",
+                    "CRITICAL": "red",
+                },
+            )
+        )
+    except ImportError:
+        handler = logging.StreamHandler()
+        handler.setFormatter(logging.Formatter(fmt=fmt, datefmt=datefmt))
+
+    logger.addHandler(handler)
+    return logger
+
+
+LOGGER = get_wideq_logger()
+
+
+def set_debug_level(level: int):
```

Comment:
Maybe `set_log_level`, because one of the options in `logging.DEBUG`?

Reply:
Yes, that makes much more sense :+1: